src/containerapp/azext_containerapp/commands.py

```diff
@@ -291,3 +291,9 @@ def load_command_table(self, args):
     with self.command_group('containerapp revision label') as g:
         g.custom_command('add', 'add_revision_label')
         g.custom_command('remove', 'remove_revision_label')
+
+    with self.command_group('containerapp env ingress', is_preview=True) as g:
+        g.custom_show_command('show', 'show_environment_ingress')
+        g.custom_command('set', 'set_environment_ingress')
```

Comment:
Refer: [Command Naming and Behavior](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md#command-naming-and-behavior)

Command `set` and `update` and `reset` don't match the guidance

> For commands which maintain child collections, the follow pairings are acceptable.
> CREATE/DELETE (same as top level resources)
> ADD/REMOVE